sql-plugin/src/main/scala/com/nvidia/spark/rapids/SparkShims.scala

```diff
@@ -100,6 +100,7 @@ trait SparkShims {
   def int96ParquetRebaseWrite(conf: SQLConf): String
   def int96ParquetRebaseReadKey: String
   def int96ParquetRebaseWriteKey: String
+  def isNegativeDecimalScaleSupportEnabled(conf: RapidsConf): Boolean
```

Comment:
This is too generic of a name. It does not indicate at all in the name, or in the comments that this is just specific to casting.